docs/rules/use-files-buffered-writer.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Use Files.newBufferedWriter
+ruleId: UseFilesBufferedWriter
+since: 3.22.0
+minJavaVersion: 7
+remediationCost: 5
+links:
+    - displayName: "Buffered I/O Methods for Text Files"
+      url: "https://docs.oracle.com/javase/tutorial/essential/io/file.html#writeBufferedStream"
+    
+description:
+    Java 7 introduced the 'java.nio.file.Files' class that contains some convenience methods for operating on files. This rule makes use of 'Files.newBufferedWriter' method for initializing 'BufferedWriter' objects to write text files in an efficient non-blocking manner.
+tags: ["Java 7", "Performance", "Old Language Constructs"]
+---
+
+# Use Files.newBufferedWriter
+
+[[toc]]
+
+## Properties
+
+<RuleProperties />
+
+## Description
+
+Java 7 introduced the [`Files`](https://docs.oracle.com/javase/7/docs/api/java/nio/file/Files.html) class that contains convenience methods for operating on files. 
+This rule makes use of the [`Files.newBufferedWriter`](https://docs.oracle.com/javase/7/docs/api/java/nio/file/Files.html#newBufferedWriter(java.nio.file.Path,%20java.nio.charset.Charset,%20java.nio.file.OpenOption...)) method for initializing [`BufferedWriter`](https://docs.oracle.com/javase/8/docs/api/java/io/BufferedWriter.html) objects to write text files in an efficient non-blocking manner.
+
+## Benefits
+
+Achieve better performance by using non-blocking I/O operations offered by the `java.nio` package.
```

Comment:
Suggestion: Add a link to `java.nio`

-> [`java.nio`](https://docs.oracle.com/javase/7/docs/api/java/nio/package-summary.html)